mod-kb-ebsco-java/src/main/resources/karate-config.js

```diff
@@ -0,0 +1,80 @@
+function fn() {
+
+  karate.configure('logPrettyRequest', true);
+  karate.configure('logPrettyResponse', true);
+
+  var retryConfig = { count: 20, interval: 30000 }
+  karate.configure('retry', retryConfig)
+
+  var env = karate.env;
+
+  // The "testTenant" property could be specified during test runs
+  var testTenant = karate.properties['testTenant'];
+
+  var config = {
+    baseUrl: 'http://localhost:9130',
+    admin: {tenant: 'diku', name: 'diku_admin', password: 'admin'},
+
+    testTenant: testTenant ? testTenant: 'testTenant',
+    testAdmin: {tenant: testTenant, name: 'test-admin', password: 'admin'},
+    testUser: {tenant: testTenant, name: 'test-user', password: 'test'},
+
+    // define global features
+    login: karate.read('classpath:common/login.feature'),
+    dev: karate.read('classpath:common/dev.feature'),
+
+    // define global functions
```

Comment:
Maybe it will better to remove function which we don't use in this module

Reply:
fixed